CRToast/CRToast.m

```diff
@@ -1049,8 +1067,18 @@ - (void)layoutSubviews {
                                       imageSize.height == 0 ?
                                       0 :
                                       CGRectGetHeight(contentFrame));
+
+    
+    if (self.toast.contentViewConfigurationBlock) {
+        self.contentView.frame = CGRectMake(CGRectGetWidth(contentFrame)-CGRectGetHeight(contentFrame)-kCRStatusBarViewNoImageRightContentInset,
+                                            statusBarYOffset,
+                                            CGRectGetHeight(contentFrame),
+                                            CGRectGetHeight(contentFrame));
```

Comment:
I did that because I figured, by reading the author's comments on similar PRs, that he wanted the view to be an accessory view that had a pre-set frame and bounds.